package.json

```diff
@@ -18,7 +18,7 @@
     "prepare": "husky"
   },
   "engines": {
-    "node": "^20.0.0"
+    "node": ">20.9.0"
```

Comment:
I think it is better to use "node": "^20.0.0" over "node": ">20.9.0" because the latter can install the latest version of Node.js, which might not be stable and could introduce breaking changes. It's better to stick with a stable version. If Node.js warns about compatibility issues or security updates, we can then evaluate and upgrade to a newer stable version as needed.

Reply:
In this case I agree in going with 20.9.0 but fixed, not ">". 20.9.0 is the node version we are running in production in Cloudflare

<img width="725" alt="Screenshot 2024-06-07 at 10 07 01" src="https://github.com/json-schema-org/website/assets/40007659/da8f880c-6daa-48f1-8c1a-3002eabedb7d">

Reply:
node `==20.9.0` in package.json and github actions now.